velox/common/file/FileSystems.cpp

```diff
@@ -140,6 +140,10 @@ class LocalFileSystem : public FileSystem {
     return filePaths;
   }
 
+  bool mkdir(std::string_view path) override {
+    return std::filesystem::create_directories(path);
```

Comment:
Hi @spershin , I see Velox often include velox/common/base/Fs.h and use the filesystem namespace there. Would want to check if we need that and need to change other occurrences in this class?

Also the generateFileDirectory() func there handles permissions. Do we also need that?
https://github.com/facebookincubator/velox/blob/abe2067bce4d1716ed3261b4927ad47e0a90d3ba/velox/common/base/Fs.cpp#L23-L34

Reply:
@gggrace14 
I see that generateFileDirectory() is only used in the tests, in few places, while FileSystem abstraction is used in production code.
generateTempFilePath() is the same, just like generateTempFolderPath().

Maybe it is worth mentioning this fact in the Fs.h